book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Kindly add `rentals` to the `att_accssor` of your book class and create a method of adding rentals. It should something close to this.
```
class Book
  attr_accessor :title, :author,:rentals

  def initialize(title, author)
    @title = title
    @author = author
    @rentals = []
  end
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
end
```

- [ ] Check out [this link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship%20def%20add_rental(person,%20date)) for more information